library/roc_consumer/src/request/request.e

```diff
@@ -0,0 +1,207 @@
+note
+	description: "Summary description for {REQUEST}."
+	date: "$Date$"
+	revision: "$Revision$"
+
+class
```

Comment:
I think I can use http_client directly, but I wanted something simpler to use.
About persistent connection, yes is not taking care of it and I agree, I will create an extension on top of HTTP client to avoid duplication.